cmd/kubent/main.go

```diff
@@ -118,12 +117,15 @@ func main() {
 	log.Info().Msg("Initializing collectors and retrieving data")
 	initCollectors := initCollectors(config)
 
-	config.TargetVersion, err = getServerVersion(config.TargetVersion, initCollectors)
+	config.TargetVersion, _ = getServerVersion(config.TargetVersion, initCollectors)
 	if config.TargetVersion != nil {
 		log.Info().Msgf("Target K8s version is %s", config.TargetVersion.String())
 	}
 
-	collectors := getCollectors(initCollectors)
+	collectors, err := getCollectors(initCollectors)
+	if err != nil {
+		os.Exit(EXIT_CODE_FOUND_ISSUES)
```

Comment:
```suggestion
		os.Exit(EXIT_CODE_FAIL_GENERIC)
```
Should return different - generic fail - exit code.

Reply:
Sure will add